### ISSUE TITLE
Prompt for only add permission on iOS

### ISSUE THREAD

**Issue by BradMclain** [yasirkula/UnityNativeGallery#40]
First of all love the plugin was just what I needed.

Would it be possible to be able to distinguish between `NSPhotoLibraryUsageDescription` and `NSPhotoLibraryAddUsageDescription` so that an app that only needs add access doesn't need to fill out `NSPhotoLibraryUsageDescription` and prompts with `NSPhotoLibraryAddUsageDescription`?

I couldn't seem to find any way to do this and `NativeGallery.SaveImageToGallery` prompts with `NSPhotoLibraryUsageDescription` as does `NativeGallery.RequestPermission`.

**Comment by yasirkula** [comment]
Relying on [this comment](https://stackoverflow.com/questions/46344159/whats-difference-between-nsphotolibraryaddusagedescription-and-nsphotolibraryus#comment79823896_46397031) by _Tien_, it seems possible to use NSPhotoLibraryAddUsageDescription only. But I don't think the app will work on older iOS devices if it is missing the NSPhotoLibraryUsageDescription.

If there is no harm in including both UsageDescription's, why not write the same description to both of them? That way, the permission dialog will always show the same description.

**Comment by BradMclain** [comment]
Yep that is what I have done at the moment but would just be _slightly_ nicer if the iOS popup was exact. It's something like `"App" would like to add to your photos ` vs `"App" would like to access your photos`. More about some minor assurance to user we are only adding vs reading from your photos.

I am also only targeting iOS11 so that part isn't an issue for me but can see why you have to worry about it.
